dashboard/test/ui/test_status.haml

```diff
@@ -1,62 +1,45 @@
-    h3 { margin-bottom: 0.1em }
-    table { width: 100% }
-    td { padding-left: 0.3em }
-    #refresh-controls { float: right; }
-    .progress-bar { margin-bottom: 0.5em }
-    .progress-bar .bar { float: left; height: 10px }
-    .progress-bar .clear-both { clear: both }
-    .SUCCEEDED, .success { background-color: #78ea78; }
-    .FAILED, .failure { background-color: #ff8888 }
-    .PENDING, .pending { background-color: #dddddd }
+  %link{:rel => :stylesheet, :type => :"text/css", :href => "test_status.css"}
   %script{src: 'https://cdn.jsdelivr.net/lodash/4.13.1/lodash.min.js'}
 %body
-  %div#refresh-controls
-    %span#last-refresh-time
-    %button#refresh-button Refresh
-    %button#auto-refresh-button Pause Auto-Refresh
-    %button#hide-succeeded-button Hide Succeeded
-  %h1= "#{type} Test Status"
-  %h2
-    %a#git-branch{href: "https://github.com/code-dot-org/code-dot-org/tree/#{git_branch}", data: {branch: git_branch}}= git_branch
-    |
-    %a#commit-hash{href: "https://github.com/code-dot-org/code-dot-org/commit/#{commit_hash}", data: {hash: commit_hash}}= commit_hash[0..7]
-    |
-    %span#start-time= start_time
-  %div#total-progress
-    %div.progress-text
-    %div.progress-bar
-      %div.success.bar
-      %div.failure.bar
-      %div.pending.bar
-      %div.clear-both
-  - browsers.sort.map do |browser|
-    %h3= browser
-    %div{id: "#{browser}-progress"}
+  .centered-column
+    %div#refresh-controls
+      %span#last-refresh-time
+      %button#refresh-button Refresh
+      %button#auto-refresh-button Pause Auto-Refresh
+      %button#hide-succeeded-button Hide Succeeded
+    %h1= "#{type} Test Status"
+    %h2
+      %a#git-branch{href: "https://github.com/code-dot-org/code-dot-org/tree/#{git_branch}", data: {branch: git_branch}}= git_branch
+      |
+      %a#commit-hash{href: "https://github.com/code-dot-org/code-dot-org/commit/#{commit_hash}", data: {hash: commit_hash}}= commit_hash[0..7]
+      |
+      %span#start-time= start_time
+    %div#total-progress
       %div.progress-text
       %div.progress-bar
         %div.success.bar
         %div.failure.bar
         %div.pending.bar
         %div.clear-both
-    %table
-      %thead
-        %tr
-          %td Browser
-          %td Feature
-          %td Status
-          %td Log
-      %tbody
-        - features.sort.map do |feature|
-          %tr{data:{browser: browser, feature: feature}}
-            %td= browser
-            %td= feature
-            %td.status
-            %td.log-link
+    - browsers.sort.map do |browser|
+      %h3= browser
+      %div{id: "#{browser}-progress"}
+        %div.progress-text
+        %div.progress-bar
+          %div.success.bar
+          %div.failure.bar
+          %div.pending.bar
+          %div.clear-both
+      %table
+        %tbody
+          - features.sort.map do |feature|
+            %tr{data:{browser: browser, feature: feature}}
+              %td= browser
+              %td= feature.sub(/features\//, '')
```

Comment:
This strips `features/` from the feature name, although we leave it as-is in the `data-feature` field on the row.